build-system/server/app-utils.js

```diff
@@ -110,67 +132,28 @@ const replaceUrls = (mode, file, hostName, inabox) => {
       /https:\/\/cdn\.ampproject\.org\/v0\/(.+?).js/g,
       hostName + '/dist/v0/$1.max.js'
     );
-    if (inabox) {
-      const filename = '/dist/amp-inabox.js';
-      file = file.replace(/<html [^>]*>/, '<html amp4ads>');
-      file = file.replace(/\/dist\/amp\.js/g, filename);
-    }
   } else if (mode == 'compiled') {
     file = file.replace(
-      /https:\/\/cdn\.ampproject\.org\/v0\.(m?js)/g,
-      hostName + '/dist/v0.$1'
-    );
-    file = file.replace(
-      /https:\/\/cdn\.ampproject\.org\/shadow-v0\.(m?js)/g,
-      hostName + '/dist/shadow-v0.$1'
-    );
-    file = file.replace(
-      /https:\/\/cdn\.ampproject\.org\/amp4ads-v0\.(m?js)/g,
-      hostName + '/dist/amp4ads-v0.$1'
-    );
-    file = file.replace(
-      /https:\/\/cdn\.ampproject\.org\/video-iframe-integration-v0\.(m?js)/g,
-      hostName + '/dist/video-iframe-integration-v0.$1'
-    );
-    file = file.replace(
-      /https:\/\/cdn\.ampproject\.org\/v0\/(.+?)\.(m?js)/g,
-      hostName + '/dist/v0/$1.$2'
+      /https:\/\/cdn\.ampproject\.org\/(.+\.m?js)/g,
+      hostName + '/dist/$1'
     );
     file = file.replace(
       /\/dist\/v0\/examples\/(.*)\.max\.(m?js)/g,
-      '/dist/v0/examples/$1.$2'
+      hostName + '/dist/v0/examples/$1.$2'
     );
     file = file.replace(
       /\/dist.3p\/current\/(.*)\.max.html/g,
       hostName + '/dist.3p/current-min/$1.html'
     );
-
-    if (inabox) {
-      file = file.replace(/\/dist\/v0\.(m?js)/g, '/dist/amp4ads-v0.$1');
-    }
-  } else if (isRtvMode(mode)) {
-    hostName = `https://cdn.ampproject.org/rtv/${mode}/`;
-    file = file.replace(/https:\/\/cdn\.ampproject\.org\//g, hostName);
-
-    if (inabox) {
-      file = file.replace(
-        /https:\/\/cdn\.ampproject\.org\/rtv\/\d{15}\/v0\.(m?js)/g,
-        hostName + 'amp4ads-v0.$1'
-      );
-    }
```

Comment:
This was incorrectly using any local version for `amp4ads-v0.js` and the specified RTV for everything else. It works after this change.